runner/options.go

```diff
@@ -82,7 +81,8 @@ func ParseOptions() *Options {
 		flagSet.StringVarP(&options.ProviderFile, "provider", "pc", defaultProviderConfigLocation, "provider configuration file"),
 		flagSet.StringVar(&options.ConfigFile, "config", defaultConfigLocation, "flag configuration file"),
 		flagSet.IntVar(&options.Timeout, "timeout", 30, "timeout in seconds"),
-		flagSet.IntVar(&options.Delay, "delay", 1, "delay between requests in seconds (0 to disable)"),
+		flagSet.IntVarP(&options.RateLimit, "rate-limit", "rl", 0, "maximum number of http requests to send per second"),
```

Comment:
I'm not sure if we should set this to 1 to maintain the original behavior of `delay`

Reply:
@Mzack9999 if the values are 0, then defaults are set directly in the session

https://github.com/projectdiscovery/uncover/blob/fd8bec3138909acf7a7e980ca6f83b9b4d903b0f/uncover/session.go#L43-L54